docs/xcm/integration/zombienet-testing.md

```diff
@@ -0,0 +1,107 @@
+
+However, using `zombienet`, users can setup a local test network where one parachain would be `Shibuya` and the other would be `Shiden` with **HRMP** channels opened between them. This is incredibly useful for testing & integration because it gives users the option for cross-chain communication between two smart-contract oriented parachains which support WASM smart contracts, amongst many other features.
+
+The following instructions will explain how to setup & configure local _Shibuya - Shiden test network_.
+
+### Basic Setup Instructions
+
+For users who already know what they are doing, please check [this](https://github.com/AstarNetwork/Astar/tree/master/third-party/zombienet) folder in `Astar` repository for _ready-to-use_ `zombienet` configurations.
+
+1. For the sake of simplicity, prepare a folder called `zombienet` into which **ALL** binaries and config files will be placed.
+
+2. Download `zombienet` binary appropriate for your operating system (or install it in any way you prefer).
+
+3. Download `polkadot` binary and `astar-collator` binary. They can be found as part of release in official [polkadot](https://github.com/paritytech/polkadot/releases) and [Astar](https://github.com/AstarNetwork/Astar/releases) repositories.
+
+4. Use the configuration file `multi_parachains.toml` which can be found [here](https://github.com/AstarNetwork/Astar/tree/master/third-party/zombienet). Make sure to check the file to get a sense of which parameters are being used.
+
+5. Start the network. This is a command example: `./zombienet-linux -p native spawn multi_parachains.toml`. However, you can start the network in your preferred way (e.g. using `podman` or `kubernetes`).
+
+6. Lots of useful information will be printed, like commands used to generate chain specifications and commands used to start the network (useful for extracting RPC ports). At the end, a table with direct links to node's `polkadot-js page` will be created. Use this to interact with chain from your browser.
+
+7. After a minute or two, block production on both `Shibuya` and `Shiden` should start. This will usually happen after **relay-chain** reaches block 11, which will trigger a new session. HRMP channels will automatically be configured between the parachains (check the configuration file).
+
+8. The test network is running and users can interract with relay chain and both parachains. It is possible to deploy EVM and WASM smart contracts, send XCM instructions and everything else possible on live chains. In addition, users have direct access to `Alice` account, which has `sudo` privileges.
+
+> We will provide automated way of performing these setup actions in the future.
+
+### Basic Cross-Chain Assets Setup Instructions
+
+After completing the previous steps, a local test network with two parachains is running.
```

Comment:
"Once the steps are completed…" vs "…will be running", IMO.

Reply:
Thanks but I prefer it like it is.